### PR TITLE
Remove some dependencies in massa-api-exports so massa-sdk have only …

### PR DESCRIPTION
**Pull request by sydhds**
…the minimum dependencies

* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
  * [ ] if part of node-launch, checked using the `resync_check` flag
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

### CODE REVIEW

**Review by aoudiamoncef — APPROVED** [review]
LGTM 👍🏻

File: massa-api-exports/src/error.rs [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
 use jsonrpsee::types::{ErrorObject, ErrorObjectOwned};
 
-use massa_consensus_exports::error::ConsensusError;
-use massa_execution_exports::ExecutionError;
+// use massa_consensus_exports::error::ConsensusError;
+// use massa_execution_exports::ExecutionError;
 use massa_hash::MassaHashError;
 use massa_models::error::ModelsError;
-use massa_protocol_exports::ProtocolError;
+// use massa_protocol_exports::ProtocolError;
```
Comment (Leo-Besancon) [inline massa-api-exports/src/error.rs:10]: Remove instead of comment

File: massa-api-exports/Cargo.toml [hunk 2]
```diff
@@ -19,10 +19,10 @@ massa_signature = {workspace = true}
 massa_time = {workspace = true}
 massa_models = {workspace = true}
 massa_final_state = {workspace = true}
-massa_consensus_exports = {workspace = true}
+# massa_consensus_exports = {workspace = true}
 massa_hash = {workspace = true}
-massa_protocol_exports = {workspace = true}
-massa_execution_exports = {workspace = true}
+# massa_protocol_exports = {workspace = true}
+# massa_execution_exports = {workspace = true}
```
Comment (Leo-Besancon) [inline massa-api-exports/Cargo.toml:25]: Remove instead of comment

**Review by Leo-Besancon — COMMENTED** [review]
LGTM with 2 cleanups

File: massa-api-exports/src/error.rs (hunk 1 above)
Comment (sydhds) [reply massa-api-exports/src/error.rs]: Removed

File: massa-api-exports/Cargo.toml (hunk 2 above)
Comment (sydhds) [reply massa-api-exports/Cargo.toml]: Rem

**Review by Leo-Besancon — APPROVED** [review]

**Comment by sydhds** [comment]
bors merge

**Comment by sydhds** [comment]
bors merge

**Pull request merged**